Source/Csla.Shared/Core/BusinessBase.cs

```diff
@@ -3599,15 +3599,8 @@ protected override void OnSetChildren(Csla.Serialization.Mobile.SerializationInf
     private bool _bypassPropertyChecks = false;
 
     /// <summary>
-    /// Is the business object currently bypassing property checks?
     /// </summary>
```

Comment:
Not in the code that I intended to commit.

However, the commit did not record the change properly.

An additional commit has been pushed for this line